phpstan.neon

```diff
@@ -15,3 +15,4 @@ parameters:
         - '#^Illuminate\\Support\\Facades\\Auth facade should not be used\.$#'
         - '#^Illuminate\\Support\\Facades\\Hash facade should not be used\.$#'
         - '#^Illuminate\\Support\\Facades\\Broadcast facade should not be used\.$#'
+        - '#^Illuminate\\Support\\Facades\\Storage facade should not be used\.$#'
```

Comment:
I'm not sure if you should ignore all of these warnings, but if you do, you can change them to a unique regular expression like the following if you want to have the specific names:

```
'#^Illuminate\\Support\\Facades\\(RateLimiter|Route|Auth|Hash|Broadcast|Storage) facade should not be used\.$#'
```

or:

```
'#^Illuminate\\Support\\Facades\\.* facade should not be used\.$#'
```

if you want to ignore all of those warning.